wire-library/wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -562,11 +562,12 @@ public TypeSpec generateEnum(EnumType type) {
         constantBuilder.addAnnotation(Deprecated.class);
       }
 
-      builder.addEnumConstant(constant.getName(), constantBuilder.build());
+      builder.addEnumConstant(nameAllocator.get(constant), constantBuilder.build());
```

Comment:
preallocate these with `newName` ?

Reply:
They're already inthere that's why I can just look them up passing the constant.
See https://github.com/square/wire/blob/master/wire-library/wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java#L226-L228